providers/dns/hurricane/internal/client.go

```diff
@@ -80,7 +81,12 @@ func (c *Client) UpdateTxtRecord(domain string, txt string) error {
 }
 
 func evaluateBody(body string, hostname string) error {
-	switch body {
+	words := strings.SplitN(body, " ", 2)
+	if len(words) == 0 {
+		return fmt.Errorf("attempt to change TXT record %s returned invalid body: %s", hostname, body)
+	}
```

Comment:
```suggestion
```

This check is useless, it's impossible to get a length of 0 with a `SplitN`:
https://play.golang.org/p/4UqWWI2lgDq

Reply:
Nice spot, thanks!